atlasdb-config/src/main/java/com/palantir/atlasdb/http/ExceptionRetryBehaviour.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.http;
+
+import com.palantir.lock.remoting.BlockingTimeoutException;
+
+import feign.RetryableException;
+
+public enum ExceptionRetryBehaviour {
+    RETRY_ON_OTHER_NODE(true, true),
```

Comment:
We will try talking to other nodes, but after 15 attempts we will fail if this is false. I've added a chunk of Javadoc on the boolean returning should... methods to clarify this.